dir.props

```diff
@@ -14,6 +14,11 @@
     <RoslynPackageName>Microsoft.Net.ToolsetCompilers</RoslynPackageName>
   </PropertyGroup>
 
+  <!-- MSBuild will set NuGetRuntimeIdentifier to be 'win' so we pre set it correctly here -->
+  <PropertyGroup>
+    <NuGetRuntimeIdentifier Condition="'$(NuGetRuntimeIdentifier)'==''">None</NuGetRuntimeIdentifier>
```

Comment:
If specifying "win" is trouble for the non-RID specific projects then we should pick a better default in the packageresolve.targets and not require this to be set in dir.props.